.github/workflows/release.yml

```diff
@@ -133,3 +259,31 @@ jobs:
         else
           publisher publish -y --location .
         fi
+
+  trim-changelog-next-on-main:
+    name: Remove released entries from CHANGELOG.next.toml on the main branch
+    if: inputs.dry_run == false && needs.get-or-create-release-branch.outputs.new_release_series == true
```

Comment:
SDK changelog entries need to end up in `aws/SDK_CHANGELOG.next.json` upon release so that the SDK changelog can be generated when it is released. It currently uses the `changelogger split` subcommand to accomplish this. Does this new release process preserve that?

Reply:
Yes - the `aws/SDK_CHANGELOG.next.json` will be on the release branch and it's generated in the previous step of the workflow.